Index.html

```diff
@@ -21,8 +21,8 @@
       as="style"
       crossorigin
       href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
-    />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
+      />
+      <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
여기는 인덴팅이 잘못 들어간 것 같습니다! 

```suggestion
    />
    <link rel="preconnect" href="https://fonts.googleapis.com" />
```